src/screens/ProfileScreen/elements/BottomSheetBio.js

```diff
@@ -25,12 +26,16 @@ const BottomSheetBio = React.forwardRef((props, ref) => {
             placeholder="Message prompt"
             keyboardAppearDelay={500}
             editable={!props.isOtherProfile}
+            maxLength={350}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a maxLength prop to the TextArea component is a good way to limit user input, but it would be better if this value was not hardcoded and instead defined as a constant at the top of the file or even better in a separate constants file. This makes it easier to manage and update these values in the future.



<!-- This is an auto-generated comment by OSS CodeRabbit -->